.github/workflows/go-test.yaml

```diff
@@ -112,6 +112,22 @@ jobs:
           kubectl -n rook-ceph scale deployment rook-ceph-osd-0 --replicas 0
           kubectl rook-ceph rook purge-osd 0 --force
 
+      - name: Restore CRD without CRName
+        run: |
+          # First let's delete the cephCluster
+          kubectl -n rook-ceph delete cephcluster my-cluster --timeout 3s || true
+
+          kubectl rook-ceph -n rook-ceph restore cephcluster
```

Comment:
Need documentation in the readme for this new command

Reply:
added, copied the docs from rook docs